app/scripts/lib/country-telephone-info.js

```diff
@@ -83,6 +83,15 @@ define((require, exports, module) => {
       prefix: '+43',
       rolloutRate: 0 // being soft launched. Testers will need to open `/sms?service=sync&country=AT`
     },
+    // Belgium
+    // https://en.wikipedia.org/wiki/Telephone_numbers_in_Belgium#Mobile_numbers
+    BE: {
+      format: formatter('+32 ${serverPhoneNumber}'),
+      normalize: ensurePrefix('+32'),
+      pattern: /^(?:\+32)?4\d{8}$/,
```

Comment:
I'm not sure whether we should check for the mobile prefix. If we do, should we allow the trunk prefix `0` before the mobile prefix? We can clean it up.

What worries me about going too far down this path is we'll have two sources of truth for validation, one on the front-end, one on the back-end. I kinda feel like we should just let the back-end handle all of this, with the caveat that the back-end will not block land-line numbers. Should that be our problem? @philbooth - thoughts?

Reply:
I reckon leave it to the back-end. At the end of the day, a determined user will probably be able to think up umpteen ways to confound our validation. If we're doing enough validation to catch obvious errors, that's good enough I reckon.